modules/auxiliary/admin/sap/sap_igs_xmlchart_xxe.rb

```diff
@@ -34,14 +34,18 @@ def initialize(info = {})
             [ 'DOS', { 'Description' => 'Denial Of Service' } ]
           ],
           'DefaultAction' => 'READ',
+          'DefaultOptions' => {
+            'VERBOSE' => true, # Enable verbose mode (show file content from remote server)
```

Comment:
This should really not be set to `true` by default since this also shows other verbose output beyond just the file content from the remote server, and most Metasploit users will not want to see this output in most cases.

Reply:
Ok, i will remove this default option.

Reply:
https://github.com/rapid7/metasploit-framework/pull/14163/commits/70e81d36613abcf9175bced0eb6b796c4f3c2980

Reply:
Thanks :) Marking resolved 👍